ci/matlab.yml

```diff
@@ -0,0 +1,57 @@
+# This is a basic workflow to help you get started with MATLAB actions
+# For more information, see: https://github.com/matlab-actions/overview
+# 
+# This workflow uses actions that are not certified by GitHub.  They are
+# provided by a third-party and are governed by separate terms of service,
```

Comment:
"third-party" is not an adjective here. Why is it hyphenated?  Suggest removing the hyphen.

Reply:
This content is copied from GitHub guidelines which contain a hyphen in third party:

> This workflow uses actions that are not certified by GitHub.
> They are provided by a third-party and are governed by
> separate terms of service, privacy policy, and support
> documentation.